### PR TITLE
[Maxwell]-Celo-Integration-with-Web3

### PR DESCRIPTION
**Pull request by maxzysparks**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.
- [x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-17-2023-Celo-Integration-with-Web3/Celo-Integration-with-Web3.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+This smart contract is based on the ERC20 standard and uses the token name "MyToken" with the symbol "MTK" as its symbol. The constructor function initializes the total supply of tokens and assigns them to the address of the contract deployer.
+
+- Next, create a new JavaScript file named `deploy.js` and add the following code:
+
+```javascript
+const ContractKit = require("@celo/contractkit");
+const Web3 = require("web3");
+const MyToken = require("./build/contracts/MyToken.json");
```
Comment (Ishanpathak1) [inline blog/03-17-2023-Celo-Integration-with-Web3/Celo-Integration-with-Web3.md:113]: this line is causing error, as you have said to find in build/contracts and also the name, make sure your  code matches the description, MyToken.sol , correct the name

**Review by Ishanpathak1 — COMMENTED** [review]
Make the changes

File: blog/03-17-2023-Celo-Integration-with-Web3/Celo-Integration-with-Web3.md (hunk 1 above)
Comment (maxzysparks) [reply blog/03-17-2023-Celo-Integration-with-Web3/Celo-Integration-with-Web3.md]: okay i just rectified it.

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**